resources/views/utils/explore/recommendationLocation.blade.php

```diff
@@ -1,10 +1,184 @@
-<div class="my-10">
-    {{-- insert code here --}}
-</div>
-
-{{--
-|--------------------------------------------------------------------------
-| SCRIPTS
-|--------------------------------------------------------------------------
---}}
-<script></script>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Slider</title>
```

Comment:
ini juga not necessarily needed, krn konsepnya title itu untuk di lihat di nama tab nya, which is ntar ga keliatan2 amat kok